lib/chronic_duration.rb

```diff
@@ -173,6 +176,14 @@ def output(seconds, opts = {})
       hours_int = hours.to_i
       hours = hours_int if hours - hours_int == 0 # if hours end with .0
       minutes = seconds = 0
+    when :hours_only
+      dividers = {
+        hours: "h", keep_zero: true
+      }
+
+      hours = hours.round(2)
+      hours_int = hours.to_i
+      hours = hours_int if hours - hours_int == 0 # if hours end with .0
```

Comment:
Clever

Reply:
Thanks! But actually, it's not mine. I copied it from another line: https://github.com/opf/openproject/blob/5ac1787fd6c441e4973032b953b141affd7d4b26/lib/chronic_duration.rb#L85-L86